js/api.js

```diff
@@ -44,7 +44,9 @@ const API = new SwaggerClient({
     url: config.api_specs,
     useJQuery: true,
     success: function() {
-        $(this).trigger('built');
+        this.readyCallbacks.forEach(callback => {
```

Comment:
But `readyCallbacks` is a list of callback ready to be called when passing through the `forEach` right?